### DISCUSSION TITLE
Baking does not match the preview

### DISCUSSION THREAD

**Discussion by Ha1fCheese** [shteeve3d/blender-wiggle-2#44] [General]
Sorry for disturbing. The addon is truly amazing. But I can’t figure out how to bake it so that the final result is like in the preview? When I look at the animation before baking, it’s one thing, and after baking it’s different.

**Comment by Ha1fCheese** [comment]
I would like to add that the main problem here is not so much a mismatch, but that the animation is a bad loop

**Comment by schroef** [comment]
Seems baking only bakes the physics it self, not the animation set on the bone. SImple test with a bone chain going from left to right, I get the physics baked, but not the motion from the bone to make the physics

Here's a simple test scene
https://github.com/user-attachments/assets/c297b78c-2a00-4abf-abf7-5a948854f54e

[reset_test_funtions-Wigglev2.zip](https://github.com/user-attachments/files/17511886/reset_test_funtions-Wigglev2.zip)

EDIT
It seems only when we use 'OVERWRITE' is the base animation of the rig also baked. Thats by design, i see native bake doing the same. Sorry, I'm not very familiar with baking

**Comment by schroef** [comment]
Quick edit and it adds now the name of the original action plus a trailing _BAKE
![added_bake_after crrect action name](https://github.com/user-attachments/assets/9983cc23-c3bc-4b16-a080-2dbf8a12b29d)
